src/scss/_components.percentage-list.scss

```diff
@@ -0,0 +1,18 @@
+.cmp-percentage-list {
+  @include body-copy;
+  list-style-type: none;
+  margin: 2rem auto;
+  padding: 0;
+  max-width: $bp-lg;
+  text-align: center;
+
+  @media (min-width: $bp-md) {
+    columns: 2;
+    text-align: left;
+    margin: 7.5rem auto;
+  }
+
+  &__percentage {
```

Comment:
I feel like you should be able to get away with your `util-bold` class here instead of a component class.